karaage/people/forms.py

```diff
@@ -92,6 +92,17 @@ class Meta:
             'address', 'country'
         ]
 
+    def clean_email(self):
+        email = self.cleaned_data['email']
+        users = Person.objects.filter(email__exact=email)
+        if users.count() > 0:
+            raise forms.ValidationError(
+                six.u(
+                    'An account with this email already exists. '
+                    'Please email %s')
+                % settings.ACCOUNTS_EMAIL)
```

Comment:
We tested and confirmed that you are correct, @brianmay. Perhaps we need to test the current email address of the user being edited against the address being saved, and if they match to accept a users.count() value of 1.